CRM/Member/Import/Parser/Membership.php

```diff
@@ -292,19 +292,16 @@ protected function getImportableFields(string $contactType = 'Individual'): arra
         ->addOrderBy('title')
         ->execute()->indexBy('name');
       foreach ($membershipFields as $fieldName => $field) {
+        if ($field['name'] === 'contact_id') {
```

Comment:
@eileenmcnaughton does this also need to be done for Participant?

Reply:
@seamuslee001 I don't know if I 100% needed this actually - it reduces confusion but I think the duplication was on Contact.id vs contact.id.

I didn't see doubles when testing https://github.com/civicrm/civicrm-core/pull/33054 for Participant (although I am going to do another pass of testing when I do one further clean up on that form)